man/man7/vdevprops.7

```diff
@@ -44,7 +44,7 @@ section, below.
 Every vdev has a set of properties that export statistics about the vdev
```

Comment:
Can you update this bit of `man/man7/zpoolconcepts.7` which you mentioned in a previous comment

```
The number of checksum errors exceeds acceptable levels and the device is
degraded as an indication that something may be wrong.
```

Reply:
fixed